### PR TITLE
Fix german translations - administration Part 1

### PR DESCRIPTION
**Pull request by tinect**
### 1. Why is this change necessary?
Currently, there are some errors in German translations.

### 2. What does this change do, exactly?
Fixes German translations in the first 7 admin-modules.
- misleading context to English source, including punctuation mark
- grammar
- spelling
- add missing translations

### 5. Checklist

- [x] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] I have written or adjusted the documentation according to my changes
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

**Comment by mitelg** [comment]
according to our [editorconfig](https://github.com/shopware/platform/blob/6.1/.editorconfig) we use two spaces in json files. could you please revert the whitespace change?

**Comment by tinect** [comment]
> according to our [editorconfig](https://github.com/shopware/platform/blob/6.1/.editorconfig) we use two spaces in json files. could you please revert the whitespace change?

@mitelg Missed that line xD Sorry. Updated Contents.

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 1]
```diff
@@ -109,22 +111,22 @@
         "buttonMappingRemoveAction": "Datenzuordnung entfernen",
         "mappingPreview": "Datenzuordnung:",
         "mappingEmptyPreview": "Für diese Datenzuordnung liegen leider keine Vorschaudaten vor.",
-        "addBlocks": "Füge Blöcke per Drag & Drop hinzu",
+        "addBlocks": "Füge Blöcke per Drag-and-drop hinzu",
```
Comment (mitelg) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]: seems to be valid https://de.wikipedia.org/wiki/Drag_and_Drop :thinking:

File: src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json [hunk 2]
```diff
@@ -17,28 +17,28 @@
...
-      "labelEmail": "E-Mail",
+      "labelEmail": "E-Mail-Adresse",
       "labelTags": "Tags"
     },
     "list": {
       "textCustomers": "Kunden",
       "textCustomersTotal": "Kein Kunde | Ein Kunde | {count} Kunden",
       "buttonAddCustomer": "Kunden anlegen",
-      "columnCustomerNumber": "Kunden-Nr.",
+      "columnCustomerNumber": "Kundennummer",
```
Comment (mitelg) [inline src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json:33]: did you check the module if the longer texts fits the column?

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json (hunk 1 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]: https://www.duden.de/rechtschreibung/Drag_and_drop 🤔

File: src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json (hunk 2 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json:33]: Didn't see any need to check, while the english one is longer with `Customer number`

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json (hunk 1 above)
Comment (marcelbrode) [reply src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]: In my opinion, we should leave the old variant. Drag & Drop as such is a defined term, which isn't wrong in the german language. The & symbol doesn't look like a foreign element in the sentence, while "and" sure does.

File: src/Administration/Resources/app/administration/src/module/sw-category/snippet/de-DE.json [hunk 3]
```diff
@@ -67,8 +67,7 @@
         "visible": "In der Navigation ausblenden",
         "imageLabel": "Anzeigebild",
         "descriptionLabel": "Beschreibung",
-                "descriptionPlaceholder": "Gib eine Beschreibung ein..."
-
+        "descriptionPlaceholder": "Gib eine Beschreibung ein..."
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-category/snippet/de-DE.json]: Please insert an empty space between "ein" and the ellipsis. Without it, it means that the word "ein" isn't complete, like being interrupted, while speaking.

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 4]
```diff
@@ -324,7 +330,7 @@
         "label": "Formular",
         "config": {
           "label": {
-            "receiverEmail": "Empfängeraddressen",
+            "receiverEmail": "Empfängeradressen",
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]: Suggestion: What do you think about `Empfänger-Adressen` to improve readability? :)

**Review by marcelbrode — CHANGES_REQUESTED** [review]

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json (hunk 1 above)
Comment (GitEvil) [reply src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]: I would like to leave that one alone. Drag & Drop is used quite consitently at the moment and honestly I don't like all these hyphens. // Würde das jetzt auch erstmal nicht anfassen wollen. Bisher ist Drag & Drop in einheitlicher Verwendung, wenigstens in den Snippets. Drag-and-drop finde ich gar nicht schön und es führt zusätzlich eine neue Uneinheitlichkeit ein (Drag großgeschrieben, drop kleingeschrieben)

File: src/Administration/Resources/app/administration/src/module/sw-category/snippet/de-DE.json (hunk 3 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-category/snippet/de-DE.json]: Arrr, this point was on my ToDo, too. Will check all occurrences

Comment (marcelbrode) [reply src/Administration/Resources/app/administration/src/module/sw-category/snippet/de-DE.json]: Nice one. Thanks for your work :+1:

Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-category/snippet/de-DE.json]: Just changed it in the related files, other modules will be fixed, when I check them these days, too. :-)

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json (hunk 1 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]: reverted to "Drag & Drop"

**Review by marcelbrode — APPROVED** [review]

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 5]
```diff
@@ -7,8 +7,9 @@
       "greeting": "Layouts",
       "createNewLayout": "Neues Layout anlegen",
       "deleteDisablesToolTip": "Das Layout ist noch mindestens einer Kategorie zugewiesen",
-      "disabledAddingBlocksToolTip": "Blöcke können nur in der Systemstandardsprache hinzugefügt werden",
-      "buttonDelete": "Löschen"
+      "disabledAddingBlocksToolTip": "Blöcke können nur in der Systemstandardsprache hinzugefügt werden.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]:
```suggestion
      "disabledAddingBlocksToolTip": "Blöcke können nur in der System-Standardsprache hinzugefügt werden.",
```

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 6]
```diff
@@ -63,7 +65,7 @@
         "contentTranslations": "Ein Layout kann aktuell nur in der Systemstandardsprache bearbeitet werden."
       },
       "helpText": {
-        "cssClassField": "Du kannst eine, oder mehrere, mit Leerzeichen getrennte, CSS-Klassen hinzufügen"
+        "cssClassField": "Du kannst eine oder mehrere, mit Leerzeichen getrennte, CSS-Klassen hinzufügen"
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json:67]:
```suggestion
        "cssClassField": "Du kannst eine oder mehrere, mit Leerzeichen getrennte, CSS-Klassen hinzufügen."
```

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 7]
```diff
@@ -111,7 +113,7 @@
         "mappingEmptyPreview": "Für diese Datenzuordnung liegen leider keine Vorschaudaten vor.",
         "addBlocks": "Füge Blöcke per Drag & Drop hinzu",
         "labelNavigation": "Navigation",
-        "lockedNotification": "Dieses Layout ist eine gesperrte Standard Vorlage die du nutzen kannst. Dieses Vorlage kann nicht angepasst werden. Wenn du diese Vorlage für eigene Bedürfnisse anpassen möchtest, kannst du es duplizieren und das Duplikat anpassen.",
+        "lockedNotification": "Dieses Layout ist eine gesperrte Standardvorlage, die du nutzen kannst. Diese Vorlage kann nicht angepasst werden. Wenn du diese Vorlage für eigene Bedürfnisse anpassen möchtest, kannst du es duplizieren und das Duplikat anpassen.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]:
```suggestion
        "lockedNotification": "Dieses Layout ist gesperrt und kann nicht angepasst werden. Solltest Du dieses Standard-Template, an Deine eigenen Bedürfnisse anpassen möchtest, kannst du es einfach duplizieren und dann das Duplikat anpassen.",
```

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 8]
```diff
@@ -121,10 +123,10 @@
         "titleMissingBlockFields": "Fehlende Konfiguration",
         "messageMissingBlockFields": "Konfiguration erforderlich. Bitte fülle alle Elemente mit Daten.",
         "titleMissingProductListing": "Fehler",
-        "messageMissingProductListing": "Speichern nicht möglich. Bitte füge einen Produktlisting-Block hinzu oder änderen den Layout-Typ.",
+        "messageMissingProductListing": "Speichern nicht möglich. Bitte füge einen Produktlisting-Block hinzu oder ändere den Layout-Typ.",
         "titlePageError": "Layout konnte nicht gespeichert werden!",
         "titleMissingSections": "Fehlende Konfiguration",
-        "messageMissingSections": "Du kannst keine leere Seite speichern. Bitte füge eine Sektion hinzu.",
+        "messageMissingSections": "Du kannst keine leere Seite speichern. Bitte füge zuerst eine Sektion hinzu.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]:
```suggestion
        "messageMissingSections": "Du kannst leere Seiten nicht speichern. Bitte füge zuerst eine Sektion hinzu.",
```

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 9]
```diff
@@ -150,8 +152,8 @@
         "elementChangeModal": "Element austauschen"
       },
       "tooltip": {
-        "blockNameField": "Vergib einen aussagekräfitgen Namen für diesen Block.",
-        "sectionNameField": "Vergib einen aussagekräftigen Namen für diese Sektion."
+        "blockNameField": "Benenne jeden Block individuell, um Deine Arbeit zu verfolgen.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]:
```suggestion
        "blockNameField": "Benenne jeden Block individuell und eindeutig, um einen guten Überblick über Deine Arbeit zu behalten.",
```

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 10]
```diff
@@ -150,8 +152,8 @@
...
+        "sectionNameField": "Benenne jede Sektion individuell, um Deine Arbeit zu verfolgen."
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]:
```suggestion
        "sectionNameField": "Benenne jede Sektion individuell und eindeutig, um einen guten Überblick über Deine Arbeit zu erhalten.."
```

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 11]
```diff
@@ -299,7 +305,7 @@
             "videoId": "Video-Link",
             "autoPlay": "Automatisch abspielen",
             "loop": "Video wiederholen",
-            "showControls": "Video-Navigation anzeigen",
+            "showControls": "Video-Steuerung anzeigen",
             "controlsColor": "Farbe der Video-Navigation ändern",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]:
```suggestion
            "controlsColor": "Farbe der Video-Steuerung ändern",
```

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json [hunk 12]
```diff
@@ -309,11 +315,11 @@
             "end": "Video abspielen bis"
           },
           "placeholder": {
-            "videoId": "Video Link eingeben ..."
+            "videoId": "Video-Link eingeben ..."
           },
           "helpText": {
-            "autoPlay": "Einige Browser verhindern das automatische Abspielen von Videos",
-            "advancedPrivacy": "Wenn du den erweiterten Datenschutzmodus aktivierst, werden von der Video-Plattform keine Informationen über die Besucher Deiner Website gespeichert, bis sie sich das Video ansehen."
+            "autoPlay": "Einige Browser verhindern das automatische Abspielen von Videos.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json]:
```suggestion
            "autoPlay": "Achtung, einige Browser verhindern das automatische Abspielen von Videos.",
```

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json [hunk 13]
```diff
@@ -112,7 +112,7 @@
         "mappingEmptyPreview": "No preview available.",
         "addBlocks": "Add blocks via drag & drop",
         "labelNavigation": "Navigation",
-        "lockedNotification": "This layout is a locked default layout template you can use for your content. You can not edit this template. If you want to adapt the layout to your needs, you can duplicate it and alter just the duplicate. ",
+        "lockedNotification": "This layout is a locked default layout template you can use for your content. You can not edit this template. If you want to adapt the layout to your needs, you can duplicate it and alter just the duplicate.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json]:
```suggestion
        "lockedNotification": "This is a locked default layout you may use for your content from the get-go. Note that you cannot edit this template! If you want to adapt the layout to your needs, just duplicate the layout and alter the duplicate.",
```

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json [hunk 14]
```diff
@@ -339,13 +339,13 @@
             "action": "Action",
             "email": "Email address*",
             "firstName": "First name*",
-            "lastName" : "Last name*",
+            "lastName": "Last name*",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json:342]: Die "required-Sternchen" müssen irgendwie anders eingefügt werden und sollten blau sein...

File: src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json (hunk 2 above)
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json:33]: Bitte schauen, ob das in die Spalte passt. Breitenmäßig. ;)

File: src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json [hunk 15]
```diff
@@ -71,18 +71,18 @@
...
       "labelPassword": "Passwort",
       "helpTextPassword": "Das Passwort kann nach dem Speichern des Kunden nur direkt vom Kunden geändert werden.",
-      "placeholderSalutation": "Anrede wählen ...",
-      "placeholderCustomerGroup": "Bitte wähle eine Kundengruppe ...",
-      "placeholderSalesChannel": "Bitte wähle einen Verkaufskanal ...",
-      "placeholderPaymentMethod": "Bitte wähle eine Zahlungsart ...",
+      "placeholderSalutation": "Anrede wählen",
+      "placeholderCustomerGroup": "Bitte wähle eine Kundengruppe",
+      "placeholderSalesChannel": "Bitte wähle einen Verkaufskanal",
+      "placeholderPaymentMethod": "Bitte wähle eine Zahlungsart",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json]: Diese Änderungen bitte komplett rückgängig machen. Placeholder sollen mit "..." dargestellt werden.

File: src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json [hunk 16]
```diff
@@ -141,7 +141,7 @@
       "columnContextOpenOrder": "Bestellung öffnen",
       "emptyTitle": "Noch keine Bestellungen",
       "emptySearchTitle": "Keine Suchergebnisse",
-      "emptySubline": "Verwalte die Bestellungen dieses Kunden."
+      "emptySubline": "Hier können Sie die Bestellungen eines Kunden verwalten"
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json]:
```suggestion
      "emptySubline": "Hier kannst Du die Bestellungen Deiner Kunden verwalten"
```
Wir nutzen im Admin "Du" als Anrede, nur in der Storefront wird "gesiezt".

File: src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/snippet/de-DE.json [hunk 17]
```diff
@@ -24,7 +24,7 @@
     "dataImport": {
       "modalTitle": "Dein Start mit Shopware 6",
       "headlineDemoData": "Demodaten benutzen",
-      "messageDemoData": "Statte Deinen Shop mit Demodaten aus, um Shopware 6 ausgiebig zu erkunden. (Die Demodaten dienen nur zu Testzwecken.)",
+      "messageDemoData": "Statte Deinen Shop mit Demodaten aus, um Shopware 6 ausgiebig zu erkunden. Die Demodaten dienen nur zu Testzwecken.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/snippet/de-DE.json]:
```suggestion
      "messageDemoData": "Statte Deinen Shop mit Demodaten aus und erkunde Shopware 6. Demodaten sollten ausschließlich zu Testzwecken verwendet werden.",
```

File: src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/snippet/de-DE.json [hunk 18]
```diff
@@ -64,7 +64,7 @@
     "recommendedPlugins": {
       "headline": "Unsere Empfehlungen",
       "modalTitle": "Plugins",
-      "message": "Hier findet Du eine Auswahl an sinnvollen Erweiterungen für Deinen Shop. Wähle zunächst eine Region aus, die für Dich relevant ist. Danach kannst Du durch verschiedene Kategorien stöbern.",
+      "message": "Hier findest Du eine Auswahl an sinnvollen Erweiterungen für Deinen Shop. Wähle zunächst eine Region aus, die für Dich relevant ist. Danach kannst Du durch verschiedene Kategorien stöbern.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/snippet/de-DE.json]:
```suggestion
      "message": "Hier findest Du eine Auswahl empfohlener Erweiterungen für Deinen Shop. Wähle zunächst die für Dich relevantesten aus. Du kannst später weiter in unterschiedlichen Erweiterungs-Kategorien stöbern.",
```

File: src/Administration/Resources/app/administration/src/module/sw-integration/snippet/de-DE.json [hunk 19]
```diff
@@ -21,13 +21,13 @@
...
-      "confirmDelete" : "Möchtest Du diese Integration wirklich löschen?",
+      "confirmDelete": "Möchtest Du diese Integration wirklich löschen?",
       "buttonApply": "Integration speichern",
       "buttonApplyEdit": "Änderungen speichern",
       "buttonCancel": "Abbrechen",
       "buttonDelete": "Löschen",
-      "buttonCreateNewApiKeys": "API-Zugangsdaten neu generieren",
-      "hintCreateNewApiKeys": "Die Neugenerierung der API-Zugangsdaten erfordert die Aktualisierung aller zugehörigen Skripte oder Anwendungen.",
+      "buttonCreateNewApiKeys": "API-Zugangsschlüssel neu generieren",
+      "hintCreateNewApiKeys": "Die Neugenerierung des API-Zugangsschlüssels erfordert die Aktualisierung aller zugehörigen Skripte oder Anwendungen.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-integration/snippet/de-DE.json]:
```suggestion
      "hintCreateNewApiKeys": "Die Neuerstellung des API-Zugangsschlüssels macht die Aktualisierung aller zugehörigen Skripte und/oder Anwendungen erforderlich.",
```

**Review by GitEvil — CHANGES_REQUESTED** [review]

File: src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json (hunk 15 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json]: Sind sie im source in diesem Fall nicht. Da auch für diesen fall anpassen? :-)

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json (hunk 6 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json:67]: https://github.com/shopware/platform/blob/071b6086b9ae4cec297048982d55abb626d3beee/src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json#L67
source hat auch keinen Punkt

File: src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json (hunk 2 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json:33]: Jepp, ist im Englischen länger

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json (hunk 14 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json:342]: Internes To-do? :-)

Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json:342]: @GitEvil ? :-)

**Comment by pantrtxp** [comment]
Hi @tinect, you still need to accept the "Contributor License Agreement" before we can go on with this PR

File: src/Administration/Resources/app/administration/src/module/sw-login/snippet/de-DE.json [hunk 20]
```diff
@@ -51,7 +51,10 @@
         "label": "E-Mail-Adresse:",
         "placeholder": "du@beispiel.de"
       },
-      "submit": "Wiederherstellungsmail senden"
+      "submit": "Wiederherstellungsmail senden",
+      "error": {
+        "passwordUpdateTitle": "Ein Fehler trat beim Ändern Deines Passworts auf:"
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-login/snippet/de-DE.json]: Hast du den Title hier geändert? Bei mir lautet er so: "passwordUpdateTitle": "Beim Aktualisieren Deines Passworts ist ein Fehler aufgetreten:" und das gefällt mir auch besser. ;)

File: src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json (hunk 14 above)
Comment (GitEvil) [reply src/Administration/Resources/app/administration/src/module/sw-cms/snippet/en-GB.json:342]: Sry, ist ja cms. Passt schon. ;)

**Review by GitEvil — APPROVED** [review]

File: src/Administration/Resources/app/administration/src/module/sw-login/snippet/de-DE.json (hunk 20 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-login/snippet/de-DE.json]: > Hast du den Title hier geändert?

Hmm? nein, es gab keine Übersetzung 🤔 Ja, dein Text ist schöner xD

**Comment by tinect** [comment]
@pantrtxp Now it's rebased onto master

**Review by GitEvil — APPROVED** [review]

**Review by GitEvil — APPROVED** [review]

**Comment by tinect** [comment]
Fixed one misleading comma :-)

File: src/Administration/Resources/app/administration/src/module/sw-login/snippet/de-DE.json [hunk 21]
```diff
@@ -1,34 +1,34 @@
...
       "headlineMain": "Hallo!",
       "subheadlineMain": "Willkommen zurück.",
-      "headlineForm": "Melde Dich in Deinem Shopware-Shop an.",
+      "headlineForm": "Melde Dich in Deinem Shopware-Shop an",
       "labelUsername": "Benutzername",
       "labelPassword": "Passwort",
-      "placeholderUsername": "Gib Deinen Benutzernamen ein ...",
-      "placeholderPassword": "Gib Dein Passwort ein ...",
+      "placeholderUsername": "Gib Deinen Benutzernamen ein",
+      "placeholderPassword": "Gib Dein Passwort ein",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-login/snippet/de-DE.json]: Diese Änderung bitte rausnehmen.

**Review by GitEvil — CHANGES_REQUESTED** [review]

**Comment by tinect** [comment]
@GitEvil Sorry, so many lines. Now ready ;-)

**Comment by pantrtxp** [comment]
Thank you for your contribution :blue_heart: :tada:

**Pull request closed without merging**